config/assets/functional.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+module.exports = {
+  client: {
```

Comment:
this is a clone of production created because that's how the configuration works right now.

Potentially we could do this. I don't see a lot of value in tackling that problem relative to other options.